String_indexOf.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>String | indexOf()</title>
+  <script>
+    var email1 = "pacoarce@gmail.com.mx";
```

Comment:
var email1 = "@.........";
var email2 = "@.........";

This will work as valid email.

You should also check that there's a word before the `@` and that theres a word after the `@` and also that theres a word after the `.`

Reply:
Añade validaciones mejores, solo estas pidiendo que el email tenga una @ y un .